packages/shared/src/lib/auth.ts

```diff
@@ -22,6 +22,7 @@ export enum AuthEventNames {
   ForgotPassword = 'forgot password',
   SubmitForgotPassword = 'submit forgot password',
   RegistrationError = 'registration error',
+  RegistrationInitialisationError = 'registration initialisation error',
```

Comment:
Let's use American form to be consistent. `Initialization` in both the key and value